src/index.js

```diff
@@ -116,43 +119,88 @@ const onMessageHandler = (target, context, message, fromSelf) => {
       })
       .catch(console.error);
   }
+  
+  if (msg === `${cmdPrefix}` + 'area') {
+    // a single area command - if the hawkuPath (in config.js) is populated, it runs like normal
+    // if there is an area manually declared, AND the path is populated, it will use the path
+    // if the area is declared, and the path is null, the static area will be used
+    if (config.commands.hawkuPath != null) {
+      const hawkuDetails = hawku.GetDetails();
+      const tabletArea = hawku.GetAreas();
+      let w = tabletArea.width; let h = tabletArea.height; let mw = tabletArea.maxwidth; let mh = tabletArea.maxheight;
+      let ar = hawkuDetails.forcedAspectRatio; let fullarea = hawkuDetails.fullArea;
```

Comment:
(And then use them below with these names)
```suggestion
      const {width, height, maxWidth, maxHeight} = tabletArea;
      const {forcedAspectRatio, fullArea} = hawkuDetails;
```